mastodon/api.py

```diff
@@ -424,6 +428,30 @@ def get_mastodon_login_url(app, login_domain, request):
     )
 
 
+def verify_client(mast_app):
+    payload = {
+        "client_id": mast_app.client_id,
+        "client_secret": mast_app.client_secret,
+        "redirect_uri": "urn:ietf:wg:oauth:2.0:oob",
+        "scope": settings.MASTODON_CLIENT_SCOPE,
+        "grant_type": "client_credentials",
+    }
+    headers = {"User-Agent": USER_AGENT}
+    url = "https://" + (mast_app.api_domain or mast_app.domain_name) + API_OBTAIN_TOKEN
```

Comment:
So by making this request, you actually made your application completely circumvent vacuuming.

Reply:
Instance admins cannot currently manage oauth apps registered by API